README.md

```diff
@@ -10,22 +10,164 @@ Inside the Arsenal Library you have different options to make amazing and quick
 ###Window Transitions
 
 Transitions:
--Fade
--Slide
--Explode
--Slide Explosion (Custom from Library)
+
+- Fade
+- Slide
+- Explode
+- Slide Explosion (Custom from Library)
 
 ###Animations
 
 Resource Animations:
--Fade In
--Fade Out
--Slide Left
--Slide Right
--Rotate
--Custom (Application Resource)
 
-##Usage
+- Fade In
+- Fade Out
+- Slide Left
+- Slide Right
+- Rotate
+- Custom (Application Resource)
+
+##In Use
+
+For Window Transitions simply call:
+```java
+  //HERE WE ARE SETTING THE ENTER TRANSITION
+  AnimationArsenal.setEnterTransition(getWindow(), AnimationArsenal
+          .getFadeTransition(new Transition.TransitionListener()
```

Comment:
getWindow().setEnterTransition() is SDK Version 21 which I'm checking in the library...